prombench/manifests/prombench/nodes_gke.yaml

```diff
@@ -8,7 +8,7 @@ cluster:
     initialnodecount: 2
     config:
       machinetype: n1-highmem-8
-      imagetype: COS
+      imagetype: COS_CONTAINERD
```

Comment:
For reference  https://cloud.google.com/kubernetes-engine/docs/deprecations/docker-containerd
![image](https://github.com/prometheus/test-infra/assets/2439858/3ae612e3-0ec5-4396-acf0-8aabfa35d3dc)